### ISSUE TITLE
Please support Flutter 3.13

### ISSUE THREAD

**Issue by fisforfaheem** [invertase/zapp.run#160]
it is too old

**Comment by fisforfaheem** [comment]
<img width="1165" alt="Screenshot 2023-10-14 at 5 48 39 PM" src="https://github.com/invertase/zapp.run/assets/35878088/92908316-6472-4594-add0-4ce6619c40f7">

**Comment by anqit** [comment]
along the same vein, dart 3 support would also be appreciated. great tool by the way, would love to see it stay up to date!

**Comment by Ehesp** [comment]
Not from a lack of trying! Dart 3 has thrown some spanners in the works which were struggling to overcome at the moment

**Comment by fisforfaheem** [comment]
Yes, thwre should be a beta version to test atleast... Its been silence
since march.. cant run new stuff

**Comment by anqit** [comment]
> Not from a lack of trying! Dart 3 has thrown some spanners in the works which were struggling to overcome at the moment

Appreciate the effort!

**Comment by Ehesp** [comment]
> Yes, thwre should be a beta version to test atleast... Its been silence since march.. cant run new stuff

We can't ship a beta to test, since we can't get it working yet. There has been many, many hours pumped into this but still hitting some complicated brick walls.

**Comment by fisforfaheem** [comment]
Ask majid hamdani for help. 😅

Atleast give a blog post about this... I thought you people abandoned the
project

**Comment by fisforfaheem** [comment]
Any progress? you may contact VERY GOOD VENTURES

**Comment by Ehesp** [comment]
Please see this issue: https://github.com/dart-lang/sdk/issues/54468

**Comment by marceloadsj** [comment]
Hey everyone, the issue mentioned by @Ehesp was closed 2 weeks ago. https://github.com/dart-lang/sdk/issues/54468
Maybe the blocker was solved and the editor can now move to the newer dart/flutter versions? :)

**Comment by fisforfaheem** [comment]
Really? @invertase/zapp.run ***@***.***> ?

**Comment by Ehesp** [comment]
Hi all,

Although the fix landed upstream in the Dart SDK did fix that issue, it unfortunately then revealed another.

We're actively working on ways to bring Zapp back to life via a different approach, using the latest Dart & Flutter SDK, in a way where there is much less risk of these breaking changes happening.

**Comment by fisforfaheem** [comment]
Thanks for the update. kindly also mention in BLOG of the website, so
people stay connected.
